test/parallel/test-event-emitter-subclass.js

```diff
@@ -47,7 +47,6 @@ assert.throws(function() {
 }, /blerg/);
 
 process.on('exit', function() {
-  assert(!(myee._events instanceof Object));
```

Comment:
Removed this as in fast path we don't have __proto__: null on the events and therefor it is an object